package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "oc",
-  "version": "0.48.0",
+  "version": "0.48.4",
```

Comment:
Can you please revert this? The version will be bumped during the publish

Reply:
Thanks @matteofigus  I have updated it. Can you please check?

Reply:
Thanks @matteofigus  I have updated it. Can you please check?